assets/js/page.ts

```diff
@@ -0,0 +1,17 @@
+export const resizePageHeight = (): void => {
```

Comment:
Minor / white space: Can we format this file with indentation of 2 spaces for consistency?